readme/BREAKING_CHANGES.markdown

```diff
@@ -1470,4 +1470,29 @@ This removal of the configuration has no replacement.
 
 ### Why was this change made?
 
-Liferay decided to not support this configuration.
\ No newline at end of file
+Liferay decided to not support this configuration.
+
+---------------------------------------
+
+## Removed support for custom SoapDescriptor
+- **Date:** 2023-June-30
+- **JIRA Ticket:** [LPS-173756](https://liferay.atlassian.net/browse/LPS-173756)
+
+### What changed?
+
+Custom `SoapDescriptor` is no longer supported.
+
+### Who is affected?
+
+This affects anyone implementing interfaces `SoapDescriptorBuilder` and `SoapDescriptor`.
+
+### How should I update my code?
+
+- Only use `soap.endpoint.name` as the key for `soap endpoint name`.
```

Comment:
@W2jx Hmmmm, this is confusing. What does this mean?

Basically "how should I update" part doesn't need to always provide a way to migrate their code relying on the changed feature. Here we remove the support of replacing the `SoapDescriptor(Builder)`, so the customer can no longer use that.

- What can the customers do to update the code besides removing their custom `SoapDescriptorBuilder`?
- If they need to configure something, how do they do that in control panel?
- And does that configuration fulfill what they originally want to do? If not, we need to state that.